src/utils/googleAnalytics.ts

```diff
@@ -0,0 +1,25 @@
+import { event, initialize, pageview } from "react-ga";
+import { REACT_APP_GA_MEASUREMENT_ID } from "secrets";
+
+export const initGoogleAnalytics = () => {
+  if (REACT_APP_GA_MEASUREMENT_ID) {
+    initialize(REACT_APP_GA_MEASUREMENT_ID, {
+      debug: true,
+    });
+    pageview(window.location.pathname + window.location.search);
+  }
+};
+
+export const logEventGoogleAnalytics = ({
+  eventCategory,
+  eventAction,
+  eventLabel,
+}: Record<string, string>) => {
```

Comment:
Those things need to be typed, not a Record<>

Reply:
fixed in 3cc41543e7a8e5a46377a4027fd29f05fa07d811